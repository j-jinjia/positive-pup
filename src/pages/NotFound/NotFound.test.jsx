import { customRender } from "../../utils/testUtils";
import NotFound from "./NotFound";

it("should render the footer component", () => {
  const { container } = customRender(<NotFound />);

  expect(container).toMatchSnapshot();
});