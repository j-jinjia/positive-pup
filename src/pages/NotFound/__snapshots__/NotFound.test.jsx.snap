// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the footer component 1`] = `
<div>
  <h1>
    404 Error Not Found
  </h1>
</div>
`;
